### DISCUSSION TITLE
Photo info

### DISCUSSION THREAD

**Discussion by PrintModz** [LycheeOrg/Lychee#1955] [Ideas]
It would be perfect if you could add information to the photos yourself and if HTML links were allowed.

Regards

**Comment by ildyria** [comment]
What do you mean? 
Could you clarify?

I am planning to make a markdown description integration in a new front-end. This is already working for the album, it will be for the photos too. (Do not expect it to be released any time soon, it is still very beta project and we don't have many people working on it).
